src/sentry/event_manager.py

```diff
@@ -393,7 +393,6 @@ def save(
         start_time: Optional[int] = None,
         cache_key: Optional[str] = None,
         skip_send_first_transaction: bool = False,
-        auto_upgrade_grouping: bool = False,
```

Comment:
The calling code does this `auto_upgrade_grouping=event_type != "transaction"`. As you look the code below you will see that it is only used when saving an _error_.